common/cont-lib/usr/share/cont-docs/70-general.txt.tpl

```diff
@@ -0,0 +1,16 @@
+General container help
+----------------------
+
+Run `docker run {% if privileged %} --privileged {% endif -%} {{ container }}{% if 'container-usage' not in cmd %} container-usage{% endif %}` to get this help.
+
+Run `docker run -ti {% if privileged %} --privileged {% endif -%} {{ container }} bash` to obtain interactive shell.
+
+Run `docker exec -ti {% if privileged %} --privileged {% endif -%} CONTAINERID bash` to access already running container.
```

Comment:
Just to be clear, the information that the container must be run as privileged container is available only in the one image (devtoolset-3-systemtap) after the files are generated.
If you suggest to remove it from this file, then it needs to be included in the documentation specific for the image. We don't have such a documentation for devtoolset-3-systemtap yet, so we'd need to create one. Is that what we need?